packages/grafana-data/src/transformations/transformers/convertFieldType.ts

```diff
@@ -147,6 +147,7 @@ export function fieldToTimeField(field: Field, dateFormat?: string): Field {
     ...field,
     type: FieldType.time,
     values: timeValues,
+    config: { custom: { format: dateFormat } },
```

Comment:
i think this is not correct.

while it's magically helpful, the tooltip for this is quite clear that this is used as a hint for the parser. it is not meant to affect the displayed output. indeed, internally it is used for _parsing_, though i dont know why, since moment's parser is super lax by default, and providing this template doesnt flip it into `strict` mode.

![](https://github.com/user-attachments/assets/8180b47e-866d-42c6-a1ee-5c6ce6197c2d)